spec/models/asp/payment_request_spec.rb

```diff
@@ -3,15 +3,48 @@
 require "rails_helper"
 
 RSpec.describe ASP::PaymentRequest do
-  subject(:asp_payment_request) { create(:asp_payment_request) }
-
   describe "associations" do
     it { is_expected.to belong_to(:asp_request).optional }
   end
 
+  describe "active?" do
+    it "returns false for state rejected" do
+      expect(create(:asp_payment_request, :rejected).active?).to be false
```

Comment:
tu peux utiliser les [predicate matchers](https://rspec.info/features/3-12/rspec-expectations/built-in-matchers/predicates/) directement :

```ruby
expect(create(:asp_payment_request, :rejected)).not_to be_active
```

etc